ftw/linkchecker/command/checking_links.py

```diff
@@ -0,0 +1,94 @@
+from AccessControl.SecurityManagement import newSecurityManager
+from zc.relation.interfaces import ICatalog
+from zope.component import getUtility
+from zope.component.hooks import setSite
+import AccessControl
+
+
+def get_plone_sites_information(app):
```

Comment:
I would `get_plone_sites` extract to a function outside of this one. This makes it possible to be accessed by other parts in the code. Right now you maybe do not use this accessibility, but perhaps a new feature in the future needs it and then it will be easier for the developer. If you extract it, IMO it would need to go into a `helpers.py` or `utils.py`.

Reply:
I'll take care of that in #18, so we can proceed with this pull request.